internal/schemautil/user_config/service_types.go

```diff
@@ -0,0 +1,1419 @@
+// Code generated by internal/schemautil/user_config/user_config_test.go; DO NOT EDIT.
+
+package user_config
+
+import (
+	schemautil "github.com/aiven/terraform-provider-aiven/internal/schemautil"
+	schema "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// ServiceTypeOpensearch is a generated function returning the schema of the opensearch ServiceType.
+func ServiceTypeOpensearch() *schema.Schema {
+	s := map[string]*schema.Schema{
+		"custom_domain": {
+			Description: "Serve the web frontend using a custom CNAME pointing to the Aiven DNS name",
+			Optional:    true,
+			Required:    false,
+			Type:        schema.TypeString,
+		},
+		"disable_replication_factor_adjustment": {
```

Comment:
I see a lot of `deprecated` words in the description. Can we add a logic that checks the description, and in case it contains a deprecated word, we also make this schema field deprecated, WDYT?